tests/test_seasonal_water_yield_regression.py

```diff
@@ -318,6 +318,30 @@ def tearDown(self):
         """Delete workspace after test is done."""
         shutil.rmtree(self.workspace_dir, ignore_errors=True)
 
+    def test_zeropadded_monthly_filenames(self):
+        """test filenames with zero-padded months in
+        _get_monthly_file_lists function
+        """
+        from natcap.invest.seasonal_water_yield.seasonal_water_yield import (
+            _get_monthly_file_lists)
+
+        n_months = 12
+
+        # Make fake paths with file names with zero-padded months
+        padded_et0_dir_list = [
+            os.path.join("/fake/path", "et0_" + str(mo).zfill(2) + ".tif")
+            for mo in range(1, n_months+1)]
+        padded_precip_dir_list = [
+            os.path.join("/fake/path", "Precip" + str(mo).zfill(2) + ".tif")
+            for mo in range(1, n_months + 1)]

```

Comment:
This is a very nice self-contained unit test!

Do you think it's worth constructing one of these lists with zero-padded numbers and one without? To demonstrate that `_get_monthly_file_lists` should accommodate both?

We could also consider a test that asserts a `ValueError` is raised in certain cases, since the function is designed with those cases in mind too.

Reply:
I believe the other tests all use non-zero-padded file names so this concept does get tested - but maybe its still good to have an explicit test for this with the new regex? Happy to add this. 

As for the fail cases - I'm thinking double zeros  (i.e. raising ValueError on precip001.tif). Did you have anything else in mind?